packages/terra-table/src/utils/focusableButton.js

```diff
@@ -0,0 +1,17 @@
+// This is a temporary file for the purpose of adding a new feature without need for refactoring across multiple components.
+// This will be removed is a separate refactor effort.
+
+const getFocusableButtonsOrHyperlinks = (parentElement) => {
+  const focusableElementSelector = 'a[href]:not([tabindex=\'-1\']), button:not([disabled]):not([tabindex=\'-1\'])';
```

Comment:
I am wondering if we need to include buttons that are created like <div> tags with a role="button", or that is out of scope?

Reply:
I believe custom elements are out of scope for this feature.

Reply:
I think you would actually have to account for this.  You might need to exclude certain types rather than being so specific.  This selector seems too restrictive.